lightning/src/ln/channel.rs

```diff
@@ -4565,9 +4565,11 @@ impl<Signer: WriteableEcdsaChannelSigner> Channel<Signer> {
 		&self.channel_type
 	}
 
-	/// Guaranteed to be Some after both ChannelReady messages have been exchanged (and, thus,
-	/// is_usable() returns true).
+	/// Guaranteed to be [`Some`] after both [`ChannelState::ChannelReady`] messages have been exchanged
+	/// (and, thus, [`is_usable()`] returns true).
 	/// Allowed in any state (including after shutdown)
+	///
+	/// [`is_usable()`]: Self::is_usable
 	pub fn get_short_channel_id(&self) -> Option<u64> {
```

Comment:
Ah wait. My bad. This will end up changing. Will let you know once I've unblocked you.